docs/terminal/learmonth.md

```diff
@@ -0,0 +1,51 @@
+---
+  title: Learmonth TCU
+---
+
+--8<-- "includes/abbreviations.md"
+
+## Positions
+
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Learmonth TCU   | Learmonth Approach   | 120.500        | LM_APP                                   |
+
+## Airspace
+
+The limits of the LM TCU are `SFC` to `F280` within 40 DME LM. This may be amended by NOTAM.
+
+By default, LM APP owns the following Restricted Areas, as detailed below:
+
+- R859A (`A045`-`A100`)  
+- R859B (`A100`-`F280`)  
+- R860A (`A015`-`A025`)  
+- R860B (`A025`-`F280`)  
+
+The above Restricted Areas are classified as Class C when LM APP is active.
+
+!!! note
+    See [VATPAC NOTAMs](https://vatpac.org/publications/notam){target=new} for active NOTAMs which may affect military operations.
+
+## Coordination
+### LM TCU / LM ADC
+
+'Next' coordination is required from LM ADC to LM TCU for all aircraft.
+
+### LM TCU / OLW 
+#### Departures
+Voiceless coordination is in place from LM TCU to OLW for aircraft:  
+Planned at or above `F280`: `Assigned F280`  
```

Comment:
If the vertical limit of the restricted area is F280, then centre can operate aircraft without coord overhead the RA at F280. This is a weird blurry line between a restricted area and a controller's airspace. I think the std assignable should be F270 in this instance